docs/release-notes.md

```diff
@@ -0,0 +1,11 @@
+---
+id: release-notes
+title: Release Notes
+sidebar_label: Release Notes
+---
+
+Release notes are stored on Github.
+
+* [PayID 1.3 - August 18, 2020](https://github.com/payid-org/payid/releases/tag/v1.3.0) - Support for self-sovereign verifiable PayID.
```

Comment:
I like this idea of calling out at least the big new feature. When available, we can link from here to the new concept page and/or tutorials, too.

Reply:
I see three slight variations on the name of this new feature in different places:

* V. PayID
* Verifiable PayID
* "self-sovereign" PayID
* Self-Sovereign Verifiable PayID

I can see why the different forms are used in different contexts but I wonder if anyone has strong opinions about what we should use as a standard in the docs. @dino-rodriguez @sappenin

Reply:
I think both "self-sovereign" and "verifiable" are important, although there is obviously some overlap. I would like some clarity on when to capitalize. "PayID Discovery" 'felt' like it should be capitalized, but I am not sure I can justify it. "self-sovereign verifiable PayID" sounds descriptive, rather than an actual name, but I could see an argument to have one single accepted phrase.